components/mysql/actions/new-row/new-row.mjs

```diff
@@ -0,0 +1,62 @@
+import mysql from "../../mysql.app.mjs";
+
+export default {
+  key: "mysql-new-row",
+  name: "New Row",
+  description: "Adds a new row. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/insert.html)",
+  type: "action",
+  version: "0.0.1",
+  methods: {
+    async insertRow({
+      table, columns = [], values = [],
+    }) {
+      if (columns.length !== values.length) {
+        throw new Error("The number of columns doesn't match the number of values");
+      }
+
+      const placehorder = values.map(() => "?").join(",");
+      const options = {
```

Comment:
I would like to suggest the same thing related to sanitizing and move the previous suggested on `delete-row.mjs` :)